2022-01-15 15:28:29,194 INFO    Thread-2  :23577 [wandb_setup.py:_flush():71] setting env: {'project': 'Amazon-Reviews', 'entity': 'ewebid', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models', 'sweep_id': 'j2xo5oqt', 'run_id': 'fyfpt359', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/sweep-j2xo5oqt/config-fyfpt359.yaml'}
2022-01-15 15:28:29,195 INFO    Thread-2  :23577 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 15:28:29,195 INFO    Thread-2  :23577 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/run-20220115_152829-fyfpt359/logs/debug.log
2022-01-15 15:28:29,196 INFO    Thread-2  :23577 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/run-20220115_152829-fyfpt359/logs/debug-internal.log
2022-01-15 15:28:29,196 INFO    Thread-2  :23577 [wandb_init.py:init():404] calling init triggers
2022-01-15 15:28:29,196 INFO    Thread-2  :23577 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 150, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.001, 'optimizer': 'adam'}
config: {}
2022-01-15 15:28:29,196 INFO    Thread-2  :23577 [wandb_init.py:init():460] starting backend
2022-01-15 15:28:29,196 INFO    Thread-2  :23577 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 15:28:29,219 INFO    Thread-2  :23577 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 15:28:29,240 INFO    Thread-2  :23577 [backend.py:ensure_launched():221] started backend process with pid: 23585
2022-01-15 15:28:29,244 INFO    Thread-2  :23577 [wandb_init.py:init():469] backend started and connected
2022-01-15 15:28:29,258 INFO    Thread-2  :23577 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 150, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.001, 'optimizer': 'adam'}
2022-01-15 15:28:29,266 INFO    Thread-2  :23577 [wandb_init.py:init():533] updated telemetry
2022-01-15 15:28:29,306 INFO    Thread-2  :23577 [wandb_init.py:init():563] communicating current version
2022-01-15 15:28:32,490 INFO    Thread-2  :23577 [wandb_init.py:init():568] got version response 
2022-01-15 15:28:32,490 INFO    Thread-2  :23577 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 15:28:32,946 INFO    Thread-2  :23577 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 15:28:37,958 INFO    Thread-2  :23577 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 15:28:37,959 INFO    Thread-2  :23577 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 15:28:37,959 INFO    Thread-2  :23577 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 15:28:37,959 ERROR   Thread-2  :23577 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-15 15:28:37,966 INFO    Thread-2  :23577 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 15:32:43,334 INFO    Thread-2  :23577 [wandb_run.py:finish():1457] finishing run ewebid/Amazon-Reviews/fyfpt359
2022-01-15 15:32:43,335 INFO    Thread-2  :23577 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-15 15:32:43,336 INFO    Thread-2  :23577 [wandb_run.py:_restore():1752] restore
2022-01-15 15:32:43,342 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 757
}

2022-01-15 15:32:43,818 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 757
}

2022-01-15 15:32:44,762 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 6904
}

2022-01-15 15:32:44,866 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18059
}

2022-01-15 15:32:44,980 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18059
}

2022-01-15 15:32:45,108 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18059
}

2022-01-15 15:32:45,215 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18059
}

2022-01-15 15:32:45,323 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18059
}

2022-01-15 15:32:45,432 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18059
}

2022-01-15 15:32:45,538 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18059
}

2022-01-15 15:32:45,642 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:45,748 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:45,854 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:45,963 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:46,067 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:46,171 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:46,274 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:47,069 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}

2022-01-15 15:32:48,200 INFO    Thread-2  :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18059
  total_bytes: 18059
}
local_info {
}

2022-01-15 15:32:49,412 INFO    Thread-2  :23577 [wandb_run.py:_append_files():2180] logging synced files
